js/src/util/scrollbar.js

```diff
@@ -13,8 +13,8 @@ const SELECTOR_FIXED_CONTENT = '.fixed-top, .fixed-bottom, .is-fixed, .sticky-to
 const SELECTOR_STICKY_CONTENT = '.sticky-top'
 
 class ScrollBarHelper {
-  constructor() {
-    this._element = document.body
+  constructor(rootElement) {
+    this._element = rootElement || document.body
```

Comment:
You miss some things here :wink: 

plus the proper tests

This [MR](https://github.com/twbs/bootstrap/pull/33665/files) will help you

Reply:
I'm not sure how I could add a test in modal (I see in the MR that we are asserting spied methods of `ScrollBarHelper`. Please let me know if you had something in mind. In the mean time, I have added a test for `ScrollBarHelper`

Reply:
I tried to fix it a bit and added the tests and functionality for scrollbar. 
I am not very sure about scrollbar modifications, so you better take a second look